server/@types/educationAndWorkPlanApi/index.d.ts

```diff
@@ -132,6 +132,22 @@ export interface paths {
     patch?: never
     trace?: never
   }
+  '/action-plans/{prisonNumber}/goals/{goalReference}/complete': {
```

Comment:
Re-imported the latest API swagger spec, which has given us the new endpoint for completing goals, plus the new types we need for this change re: notes